docs/MO_DG/prepare_model/Getting_performance_numbers.md

```diff
@@ -1,28 +1,28 @@
 
-> **NOTE**: Some image pre-processing can be baked into the IR and accelerated accordingly. For more information, refer to [Embedding the Preprocessing](Additional_Optimizations.md). Also consider [Runtime Optimizations of the Preprocessing](../../optimization_guide/dldt_deployment_optimization_common).
+> **NOTE**: Some image pre-processing can be baked into the IR and accelerated accordingly. For more information, see the [Embedding the Preprocessing guide](Additional_Optimizations.md). Additional information are also present in the [Runtime Optimizations of the Preprocessing guide](../../optimization_guide/dldt_deployment_optimization_common.md).
 
-## Tip 2. Getting Credible Performance Numbers 
+## Tip 2: Getting Credible Performance Numbers 
 
-You need to build your performance conclusions on reproducible data. Do the performance measurements with a large number of invocations of the same routine. Since the first iteration is almost always significantly slower than the subsequent ones, you can use an aggregated value for the execution time for final projections:
+Performance conclusions should be build upon reproducible data. As for the performance measurements, they should be done with a large number of invocations of the same routine. Since the first iteration is almost always significantly slower than the subsequent ones, you can use an aggregated value for the execution time for final projections:
 
 -	If the warm-up run does not help or execution time still varies, you can try running a large number of iterations and then average or find a mean of the results.
 -	For time values that range too much, consider geomean.
--   Beware of the throttling and other power oddities. A device can exist in one of several different power states. When optimizing your model, for better performance data reproducibility consider fixing the device frequency. However the end to end (application) benchmarking should be also performed under real operational conditions.
+-  Beware of the throttling and other power oddities. A device can exist in one of several different power states. When optimizing your model consider fixing the device frequency for better performance data reproducibility. However, the end to end (application) benchmarking should also be performed under real operational conditions.
 
-## Tip 3. Measure Reference Performance Numbers with OpenVINO's benchmark_app 
+## Tip 3: Measure Reference Performance Numbers with OpenVINO's benchmark_app 
 
-To get performance numbers, use the dedicated [Benchmark App](../../../samples/cpp/benchmark_app/README.md) sample which is the best way to produce the performance reference.
-It has a lot of device-specific knobs, but the primary usage is as simple as: 
+To get performance numbers, use the dedicated [Benchmark App](../../../samples/cpp/benchmark_app/README.md) sample as it is the best soultion to produce the performance reference.
```

Comment:
```suggestion
To get performance numbers, use the dedicated [Benchmark App](../../../samples/cpp/benchmark_app/README.md) sample, as it is the best solution to produce the performance reference.
```
If you can substitute phrase "as it is" with "because it is", then it is preceded with a comma :)